ros_buildfarm/templates/status/blocked_source_entries_page_job.xml.em

```diff
@@ -0,0 +1,110 @@
+<project>
+  <actions/>
+  <description>Generated at @ESCAPE(now_str) from template '@ESCAPE(template_name)'</description>
+  <keepDependencies>false</keepDependencies>
+  <properties>
+@(SNIPPET(
+    'property_log-rotator',
+    days_to_keep=100,
+    num_to_keep=100,
+))@
+@(SNIPPET(
+    'property_job-priority',
+    priority=20,
+))@
+@(SNIPPET(
+    'property_requeue-job',
+))@
+  </properties>
+@(SNIPPET(
+    'scm_git',
+    url='https://github.com/ros-infrastructure/ros_buildfarm',
```

Comment:
Use variables as in the other templates for this.

Same for the `branch_name`.

Reply:
Using variables for url and branch name on both this page and the blocked_releases page in 9501fce